storybook/storybook-docs/src/typography.stories.mdx

```diff
@@ -1,84 +1,160 @@
+Er zijn verschillende manieren om tekst te accentueren.
+Hierin hebben we de volgende keuzes gemaakt.
 
-## Uitlijning
+### Gewicht
 
-Het aantal karakters op een regel is maximaal 80.
-En de tekst wordt altijd links uitgelijnd.
-Dit is vanwege leesbaarheid.
+Online gebruiken we 2 gewichten: ‘regular’ en ‘extra bold’.
 
-Uitzonderingen:
+<Typeset
+  fontFamily="Amsterdam Sans, Arial, sans-serif"
+  fontSizes={[24]}
+  fontWeight={400}
+  sampleText="Jouw typograaf biedt mij zulke exquise schreven"
+/>
+
+Tekst wordt vetgedrukt in titels en decoratieve citaten.
+
+<Typeset
+  fontFamily="Amsterdam Sans, Arial, sans-serif"
+  fontSizes={[24]}
+  fontWeight={800}
+  sampleText="Jouw typograaf biedt mij zulke exquise schreven"
+/>
+
+### Stijl
 
-- Op knoppen worden labels gecentreerd.
-- In tabellen kan de inhoud van kolommen, afhankelijk van de inhoud, ook gecentreerd of rechts uitgelijnd worden.
+Gebruik de cursieve weergave uitsluitend om fragmenten in tekst te accentueren.
+Cursieve tekst is minder goed leesbaar dan reguliere of vetgedrukte tekst.
 
-## Vormgeving
+Pas het niet toe op hele tekstblokken.
+Er zijn betere manieren om lange teksten nadruk te geven.
 
-Vermijd zo veel mogelijk cursieve teksten, deze zijn niet goed te lezen op sommige beeldschermen.
-Gebruik ze uitsluitend om tekst te accentueren, niet voor hele tekstblokken.
+<Typeset
+  fontFamily="Amsterdam Sans, Arial, sans-serif"
+  fontSizes={[24]}
+  sampleText="Jouw typograaf biedt mij zulke exquise schreven"
+  style={{ fontStyle: "italic" }}
+/>
 
-Vermijd het onderlijnen van tekst.
-Reserveer het onderlijnen van tekst voor interactieve links.
+### Onderstreping
+
+Uitsluitend de tekst van een hyperlink wordt onderstreept.
+Rond staartletters wordt de lijn onderbroken voor betere leesbaarheid.
+
+Onderstreepte tekst zonder hyperlink geeft toch de suggestie van een link.
+Dat is verwarrend en daarom niet toegestaan.
+Gebruik een andere manier om te accentueren.
+
+<Typeset
+  fontFamily="Amsterdam Sans, Arial, sans-serif"
+  fontSizes={[24]}
+  sampleText="Jouw typograaf biedt mij zulke exquise schreven"
+  style={{ color: "#004699", textDecoration: "underline" }}
+/>
+
+### Overhang
 
 De ruimte tussen letters (kerning) is geoptimaliseerd in het lettertype.
 Het is niet toegestaan om de kerning aan te passen.
 
-## Woorden afbreken
+## Andere eigenschappen
+
+### Contrast
+
+Het contrast tussen tekst en achtergrondkleur moet voldoen aan de laatste wettelijke toegankelijkheidseisen van [de WCAG, niveau AA](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html).
 
-De Nederlandse taal kent veel lange woorden, zoals 'gemeenteraadsverkiezingen'.
+De meeste tekst in het design system is puur zwart op puur wit en dat voldoet daar ruim aan.
+Op de meeste donkere achtergronden maken we tekst wit; ook dat biedt voldoende contrast.
+Voor sommige achtergrondkleuren is ook zwarte tekst goed leesbaar.
+Raadpleeg de betreffende componenten voor specifieke richtlijnen.
```

Comment:
This is a bit vague. Shouldn't we [link to Figma](https://www.figma.com/file/9IGm6IdPUYizBNGsUnueBd/Amsterdam-Design-System?node-id=2%3A294&mode=dev) here? There's an overview of what text color goes with what background color there.